dev/AppNotifications/AppNotificationUtility.cpp

```diff
@@ -265,27 +265,46 @@ void Microsoft::Windows::AppNotifications::Helpers::RegisterAssets(std::wstring
     std::wstring iconFilePath{};
 
     wil::unique_prop_variant propVariantIcon;
-    // Throw in case of failure, since Icon is mandatory and we don't have a fallback!
-    THROW_IF_FAILED(propertyStore->GetValue(PKEY_AppUserModel_RelaunchIconResource, &propVariantIcon));
-    THROW_HR_IF_MSG(E_UNEXPECTED, propVariantIcon.vt == VT_EMPTY, "You must specify an app icon before calling Register().");
+    // In case there is no icon asset, do not throw. Set a default icon instead.
+    LOG_IF_FAILED(propertyStore->GetValue(PKEY_AppUserModel_RelaunchIconResource, &propVariantIcon));
```

Comment:
I'm not sure I understand this. Get nervous when i see code that swallows errors. Why are we not throwing? @danielayala94

Reply:
Same concern for DisplayNameResource above.